src/sql/connection.py

```diff
@@ -380,6 +380,7 @@ def set(cls, descriptor, displaycon, connect_args=None, creator=None, alias=None
                     cls.current = existing
                 # if just switching connections
                 elif existing and alias is None:
+                    display.message(f"Switching to connection {descriptor}")
```

Comment:
I think the connection url can also be appended to the message . what do you think @edublancas

Reply:
I have added this condition now!